### PR TITLE
Rename Clusters to Tenants

### PR DESCRIPTION
**Pull request by dvaldivia**

### CODE REVIEW

File: restapi/configure_m3.go.orig [hunk 1]
```diff
@@ -0,0 +1,147 @@
+// This file is safe to edit. Once it exists it will not be overwritten
+
```
Comment (cesnietor) [inline restapi/configure_m3.go.orig]: remove this file

**Review by cesnietor — CHANGES_REQUESTED** [review]
please remove all .orig files

File: restapi/configure_m3.go.orig (hunk 1 above)
Comment (dvaldivia) [reply restapi/configure_m3.go.orig]: done

**Review by Alevsk — DISMISSED** [review]

**Review by cesnietor — CHANGES_REQUESTED** [review]
you also need to delete all the operations/admin_api cluster related files and the models/...cluster...* related files since those are not deleted on swagger generation.

**Comment by dvaldivia** [comment]
@cesnietor done

File: restapi/tenants.go [hunk 2]
```diff
@@ -195,6 +195,26 @@ func getClusterCreatedResponse(params admin_api.CreateClusterParams) error {
...
+			Requests: corev1.ResourceList{
+				corev1.ResourceStorage: volumeSize,
+			},
+		},
+	}
+
+	if params.Body.VolumeConfiguration.StorageClass != "" {
+		volTemp.StorageClassName = &params.Body.VolumeConfiguration.StorageClass
+	}
+
```
Comment (cesnietor) [inline restapi/tenants.go]: this seems not part of this PR

**Review by cesnietor — APPROVED** [review]
LGTM

**Pull request merged**